app/src/app/views/admin-page/org-settings-page/org-settings-page.component.html

```diff
@@ -0,0 +1,108 @@
+                    objects will likewise be marked with your organization as the modifier.
+                </p>
+                <p>
+                    Organization identity will be shown to anyone downloading collections that you publish. Updates to
+                    your organization identity will automatically update the contents of your knowledge base, but
+                    published collections will not be affected.
+                </p>
+            </div>
+        </div>
+        <div *ngIf="organizationIdentity else loading" class="identity-edit grid">
+            <hr>
+            <div class="row">
+                <div class="col narrow vertical-center identity-icon">
+                    <app-identity-property [config]="{
+                    'object': {'identity': organizationIdentity},
+                    'field': 'identity'
+                }"></app-identity-property>
+                </div>
+                <div class="col">
+                    <mat-form-field appearance="outline">
+                        <mat-label>organization name</mat-label>
+                        <input matInput required [(ngModel)]="organizationIdentity.name">
+                    </mat-form-field>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col">
+                    <mat-form-field appearance="outline">
+                        <mat-label>organization description</mat-label>
+                        <input matInput [(ngModel)]="organizationIdentity.description">
+                        <mat-hint>this field can be left blank if desired</mat-hint>
+                    </mat-form-field>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col center">
+                    <div>
+                        <button mat-stroked-button (click)="saveIdentity()" class="text-label"
+                                [disabled]="organizationIdentity.name.length == 0">save
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="view-page grid spacing-16">
+        <div class="row">
+            <div class="col">
+                <h2>Namespace Settings</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col">
+                <p>Assigning a unique namespace will uniquely identify any objects you create and add to or modify in the knowledge base.</p>
+                <p>The namespace prefix will a short term in all upper-case, appended to the beginning of every new object ID.
+                    The namespace range is a 4-digit number that will specify the lower-bound to start generating new object IDs from.</p>
+            </div>
+        </div>
+        <div class="identity-edit grid spacing-12">
```

Comment:
In case the namespace settings are taking a long time to load, there should be a conditional here to show a progress spinner until the data is loaded (similar to the organization identity loading overlay).